internal/pikamon/util/global.go

```diff
@@ -0,0 +1,16 @@
+package util
```

Comment:
util packages in Go are a bit of an anti-pattern. The typical pattern is for types, constants, and vars that are shared by multiple packages to go in either a `common` package with all of them, or a package for each (i.e. a `types` package, a `constants` package, and a `vars` package. Common functions are a bit more of a wild west, and served on a case by case basis.

Reply:
The pattern my team at work has been using for common functions is to have a core.go file at a level in the package hierarchy where it is accessible to everything that needs it. For us, that would be a core.go file in `internal/pikamon`, so the functions would be accessed as `pikamon.Thing()`.

Reply:
Created this issue which I will start working on next
- https://github.com/Jac0bDeal/pikamon/issues/24